src/common/components/organisms/FidgetSettingsEditor.tsx

```diff
@@ -1,24 +1,24 @@
-import React, { useState } from "react";
-import {
-  FidgetSettings,
-  FidgetProperties,
-  FidgetFieldConfig,
-} from "@/common/fidgets";
-import BackArrowIcon from "../atoms/icons/BackArrow";
-import { FaTrashCan, FaTriangleExclamation } from "react-icons/fa6";
 import { Button } from "@/common/components/atoms/button";
 import {
   Tabs,
+  TabsContent,
```

Comment:
From what I can tell, this is all the same content, but reordered. Can you clean this up to either make it clear that nothing has changed, or highlight the changes in a real way?

Reply:
yeah sorry about that, I have this vscode settings that when I save a file it reorder the imports. and as it was my first time exploring the code (with thelaunching rush) I have autosaved some files after that plugin I use reorganize it. ups

Reply:
the good news is that is in alphabetical order now ehehe